### PR TITLE
LPS-185522 | Test Fix | Master

### PR DESCRIPTION
**Pull request by Luizcarlosqueiroz**
**Ticket:** [LPS-185522](https://issues.liferay.com/browse/LPS-185522)

### CODE REVIEW

**Comment by Luizcarlosqueiroz** [comment]
Test passing at CI https://github.com/liferay-bpm-qa/liferay-portal/pull/859

**Comment by Luizcarlosqueiroz** [comment]
ci:test:sf

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByAssignee.testcase [hunk 1]
```diff
@@ -32,6 +32,7 @@ definition {
 	@description = "LPS-97666 - Verify that the names inside the steps filter component from Performance by Assignee card are breaking into another line when the component width exceeds 240px"
 	@priority = 3
 	test AssigneeNamesBreaksIntoAnotherLine {
+		property portal.acceptance = "defaulttest";
```
Comment (patriciajeanperez) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByAssignee.testcase:35]: @Luizcarlosqueiroz Looks good! Go ahead and forward a PR without this property. Thank you!

Comment (Luizcarlosqueiroz) [reply modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByAssignee.testcase]: Oh, I forgot to remove it. Thanks!

**Pull request closed without merging**